configuration/src/main/java/io/camunda/configuration/Backup.java

```diff
@@ -57,6 +57,14 @@ public class Backup {
    */
   private Duration incompleteCheckTimeout = Duration.ofMinutes(5);
 
+  /**
+   * Enable or disable secondary storage backup endpoint.
+   *
+   * <p>Note: This is an internal configuration, enabled when operate and/or tasklist webapps are
+   * enabled, and also enabled by the standalone backup application.
+   */
+  private boolean secondaryStorageEnabled = false;
```

Comment:
:question: Do you need a new property for this? Or can you use `camunda.data.secondary-storage.type` == `NONE` for this?